test/plug_test.exs

```diff
@@ -15,23 +16,22 @@ defmodule Sentry.PlugTest do
   end
 
   test "exception makes call to Sentry API" do
-   bypass = Bypass.open
-   Bypass.expect bypass, fn conn ->
-     {:ok, body, conn} = Plug.Conn.read_body(conn)
-     assert body =~ "RuntimeError"
-     assert body =~ "ExampleApp"
-     assert conn.request_path == "/api/1/store/"
-     assert conn.method == "POST"
-     Plug.Conn.resp(conn, 200, ~s<{"id": "340"}>)
-   end
-   Application.put_env(:sentry, :dsn, "http://public:secret@localhost:#{bypass.port}/1")
-   Application.put_env(:sentry, :included_environments, [:test])
-   Application.put_env(:sentry, :environment_name, :test)
```

Comment:
The config for the project is set to these values in https://github.com/getsentry/sentry-elixir/blob/master/config/config.exs and https://github.com/getsentry/sentry-elixir/blob/master/config/test.exs